indico/modules/events/registration/models/registrations.py

```diff
@@ -645,7 +646,9 @@ def get_picture_attachments(self):
             if data.storage_file_id is None:
                 continue
             with data.open() as f:
-                attachment = MIMEImage(f.read())
+                if not (thumbnail_bytes := resize_uploaded_registration_picture(f, 120)):
+                    continue
+                attachment = MIMEImage(thumbnail_bytes.read(), _subtype='jpeg')
```

Comment:
```suggestion
                attachment = MIMEImage(thumbnail_bytes.read(), 'jpeg')
```

I don't think that one is meant to be passed as a kwarg, considering it's not kw-only and underscored.